src/lib/_common.scss

```diff
@@ -159,6 +163,10 @@ $header-height: 48px;
     }
 }
 
+.bp.bp-loaded .bp-content {
+    transition: left 300ms cubic-bezier(.4, 0, .2, 1);
```

Comment:
Could this affect anything when the thumbnail sidebar is off?

Reply:
Yeah, that's why I added the `.bp-loaded` specificity because it affected the loading view, but I'll hunt around to check again